pkg/kritis/apis/kritis/v1beta1/types.go

```diff
@@ -54,3 +54,26 @@ type ImageSecurityPolicyList struct {
 
 	Items []ImageSecurityPolicy `json:"items"`
 }
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type AttestationAuthority struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	NoteReference        []string `json:"noteReference"`
```

Comment:
why are these string slices? The example shows a single string.

Reply:
fixed